benchmark_runner/common/oc/oc.py

```diff
@@ -276,15 +276,27 @@ def get_long_uuid(self, workload: str):
             f"{self.__cli} -n {environment_variables.environment_variables_dict['namespace']} get benchmark/{workload} -o jsonpath={{.status.uuid}}")
         return long_uuid
 
+    def get_major_version(self):
```

Comment:
Name this `get_ocp_major_version` and document it as "Return OCP major version".   You might also want to sanity check the answer.